restli-common/src/main/java/com/linkedin/restli/common/ContentType.java

```diff
@@ -55,6 +55,7 @@
   private static final JacksonLICORStreamDataCodec
       LICOR_BINARY_STREAM_DATA_CODEC = new JacksonLICORStreamDataCodec(R2Constants.DEFAULT_DATA_CHUNK_SIZE, true);
   private static final ProtobufDataCodec PROTOBUF_DATA_CODEC = new ProtobufDataCodec();
+  private static final ProtobufDataCodec LI_PROTOBUF_DATA_CODEC = new ProtobufDataCodec(null, true);
```

Comment:
Nit: For open source, LI in the name or header has too much trace on the contributing company, why do we need to use LI in the header name, can we name it based on functionality?

Reply:
I agree with removing LI. I am just gonna call it protobuf v2.